### PR TITLE
fix(dashboard): `NodeJS.Global` type missing

### PR DESCRIPTION
**Pull request by aymene01**
## About

Just fixing the NodeJS.Global type missing

### CODE REVIEW

File: packages/dashboard/lib/prisma.ts [hunk 1]
```diff
@@ -1,11 +1,8 @@
 import { PrismaClient } from '@prisma/client';
 
-// @ts-expect-error NodeJS.Global does not exists
-interface CustomNodeJsGlobal extends NodeJS.Global {
+declare const global: typeof globalThis & {
```
Comment (QuiiBz) [inline packages/dashboard/lib/prisma.ts:3]: Smart! Could you duplicate the same inside `lib/redis.ts` and `lib/s3.ts`?

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: packages/dashboard/lib/prisma.ts (hunk 1 above)
Comment (aymene01) [reply packages/dashboard/lib/prisma.ts:3]: Done !

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**